src/keyszer/config_api.py

```diff
@@ -166,8 +166,7 @@ def _to_keystrokes(ctx):
         combo_list = []
         for c in s:
             if ord(c) > 127:
-                unicode_keystrokes(ord(c))(ctx)    # extra "()" updates global variable
-                combo_list.extend(unicode_combo_list)
+                combo_list.extend(unicode_keystrokes(ord(c))(ctx))
```

Comment:
Oh, I was imagining appending the inner function (and letting the processor call it), but I suppose this works also since we're doing all the work on the fly in both cars... so if this works, then ok...

Reply:
Uh, yeah... I'm not sure what you mean here, but this is just ending up returning the object that returns the combo_list from the Unicode function, soooo... seems to be accomplishing the same thing as when I was storing the Unicode combo_list in the global list.

Reply:
If I understand the overlord's explanation of how to call an inner function directly in Python, I'm pretty sure that's already what's happening here (and why the extra "(ctx)" tacked on the end is necessary) since the Unicode function is now returning a function. Which is being called. 

Unless I'm still not grasping the concepts here.

Reply:
If I get what's happening here, this is essentially doing `outer_func()()`, thus calling on the inner function, since `inner` is defined as `outer()`, then you call `inner()` which adds an additional "()" call. Which as far as I can tell is exactly what the code does to call on `unicode_keystrokes`. 

```py
def outer_func():
    def inner_func():
        print("Hello from inner_func!")
    
    return inner_func

inner = outer_func()
inner() # This will call the inner function
```

Reply:
So I just finally explained to myself why the additional "(ctx)" was necessary for it to produce results. LOL. 

That's quite amusing.  🤏🏽 🍵 🧐

Reply:
I was suggesting:

```
combo_list.extend(unicode_keystrokes(ord(c)))
```

Which would put the inner function on the command list NOT run it - which puts the array it builds onto the command list... instead it would be run later when the command processor was looping over the command loop... but I'm not sure there is a real advantage in one way or the other.

Reply:
I see. That ended up not working when I was just trying to populate the global list. But maybe it would work if it was just an object that would then get run later... 

Trying... And... 

```py
TypeError: 'function' object is not iterable
```

Like it's trying to use it as a function object rather than run it and obtain the combo_list inside it? Dunno.

Reply:
Oh you wouldn't use `extend`, it's not an array anymore... you'd use `append`.

Reply:
Huh. Yeah, it works with `append()`. Seems to have the same effect, adapting to CapsLock just fine. 

```py
                combo_list.append(unicode_keystrokes(ord(c)))
```

Looks cleaner, at least. No idea if there is some other advantage. But I'm fine with updating it.